src/test/java/com/karankumar/bookproject/backend/service/BookServiceTest.java

```diff
@@ -18,22 +18,15 @@
 package com.karankumar.bookproject.backend.service;
 
 import com.karankumar.bookproject.annotations.IntegrationTest;
-import com.karankumar.bookproject.backend.entity.Author;
-import com.karankumar.bookproject.backend.entity.Book;
-import com.karankumar.bookproject.backend.entity.BookGenre;
-import com.karankumar.bookproject.backend.entity.CustomShelf;
-import com.karankumar.bookproject.backend.entity.PredefinedShelf;
-import com.karankumar.bookproject.backend.entity.RatingScale;
 import com.karankumar.bookproject.backend.entity.Tag;
+import com.karankumar.bookproject.backend.entity.*;
```

Comment:
Could you please revert this change? As per our style guide, we are not using wildcard imports